README.md

```diff
@@ -1,7 +1,63 @@
-## Build dependencies ##
+## Overview ##
+
+Libnvcrypt is a helper library to store 128 bit extra random data in the TPM
```

Comment:
Let's not talk about "128 bit extra random", but maybe rather "a second password that is needed for LUKS encryption".